src/FsToolkit.ErrorHandling/List.fs

```diff
@@ -106,7 +106,12 @@ module List =
             | Ok _, Error errs
             | Error errs, Ok _ -> traverseValidationA' (Error errs) f xs
 
-    let traverseValidationA f xs = traverseValidationA' (Ok []) f xs
+    let traverseValidationA f xs =
```

Comment:
I think there's a similar fix for this type of behavior where you can keep it in `traverseValidationA'`

https://github.com/demystifyfp/FsToolkit.ErrorHandling/blob/31784fa747d7a7dfd5a03c61f8a3ef436c117d51/src/FsToolkit.ErrorHandling/List.fs#L57

Reply:
Yes, that is simpler.